l/lit/Lit_ubi_8.5.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
```

Comment:
Why is the folder and file named lit*?
Please avoid using capital letters in filename.

Reply:
Named as lit because, script focuses only on lit from llvm-project.